bindings/go/evmc/evmc.go

```diff
@@ -147,6 +147,8 @@ const (
 	Constantinople   Revision = C.EVMC_CONSTANTINOPLE
 	Petersburg       Revision = C.EVMC_PETERSBURG
 	Istanbul         Revision = C.EVMC_ISTANBUL
+	Berlin           Revision = C.EVMC_BERLIN
```

Comment:
Hmm, isn't this a bugfix which should be released as 8.0.1?

Reply:
It is, but we don't need to fix it proactively.